ang/civicase/case/details/directives/case-details.directive.js

```diff
@@ -200,9 +202,8 @@
         delete ($scope.item.tag_id);
         _.assign($scope.item, formatCaseDetails(data));
         $scope.allowedCaseStatuses = getAllowedCaseStatuses($scope.item.definition);
-
-        $scope.$broadcast('updateCaseData');
```

Comment:
Why are we removing these lines?

Reply:
Thanks for spotting this @deb1990 ! this was going to be a regression, it was not meant to be included. I probably replaced it when trying to move the line around. Strange thing is that there are no tests for these events, so tests were passing fine. I don't have enough time to add these tests, but will keep them in the radar for the future.